infrastructure/general/setup-full.md

```diff
@@ -0,0 +1,366 @@
+to be able to look at past on-chain data.
+
+A *full node* prunes historical states: all finalized blocks' states older than a configurable
+number except the genesis block's state. This is 256 blocks from the last finalized one by default.
+A pruned node this way requires much less space than an archive node.
+
+A full node could eventually rebuild every block's state without additional information and become
+an archive node. This still needs to be implemented at the time of writing. If you need to query
+historical blocks' states past what you pruned, you must purge your database and resynchronize your node,
+starting in archive mode. Alternatively, you can use a backup or snapshot of a trusted source to
+avoid needing to sync from genesis with the network and only need the states of blocks past that
+snapshot.
+
+Full nodes allow you to read the current state of the chain and to submit and validate extrinsics
+directly on the network without relying on a centralized infrastructure provider.
+
+Another type of node is a *light node*. A light node has only the runtime and the current state
+but doesn't store past blocks and so cannot read historical data without requesting it from a node
+that has it. Light nodes are useful for resource-restricted devices. An interesting use-case of
+light nodes is a browser extension, which is a node in its own right, running the runtime in WebAssembly
+format, as well as a full or light node that is completely encapsulated in WebAssembly and can be
+integrated into web apps: [Smoldot](https://github.com/smol-dot/smoldot.){target=_blank}
+
+!!!note Substrate Connect
+    [Substrate Connect](https://github.com/paritytech/substrate-connect){target=_blank} provides a way to interact with
+    substrate-based blockchains in the browser without using an RPC server. It is a light node that runs
+    entirely in JavaScript. Substrate Connect uses a
+    [Smoldot Wasm light client](https://github.com/paritytech/smoldot){target=_blank} to securely connect to the
+    blockchain network without relying on specific third parties. Substrate Connect is available on Chrome
+    and Firefox as a [browser extension](https://substrate.io/developers/substrate-connect/){target=_blank}.
+
+
+<!--separates content from instructions-->
+
+---
+
+<!--setup instructions differ per os, presented in tabs-->
+
+## Setup Instructions
+
+This isn't recommended if you're a validator. Please see the [secure validator setup](maintain-guides-secure-validator.md) if you are running validator.
+
+!!!note 
+    The bash commands that are provided to run against **your node** use `Polkadot` as the
+    default chain
+
+    Use the `--chain` flag if you follow the setup instructions to setup a `Kusama` node. For example:
+
+    ```bash
+    polkadot --name "Your Node's Name" --chain kusama
+    ```
+
+=== "macOS"
+
+    - Install Homebrew within the terminal by running:
```

Comment:
There are some formatting issues with this list items. Also, you could use a numeric list instead of `-`